services/watchcat/config.in

```diff
@@ -1 +1,4 @@
-dep_bool_editor "Watch IO changes (and react)" WATCHCAT_SUPPORT "services/watchcat/config" $ECMD_SENDER_SUPPORT $PORTIO_SUPPORT
+dep_bool_menu "Watch IO changes (and react)" WATCHCAT_SUPPORT $ECMD_SENDER_SUPPORT $PORTIO_SUPPORT
+editor "Configuration" WATCHCAT_SUPPORT "services/watchcat/config"  
```

Comment:
I'd better call the option ```Edit configuration``` and include a non-empty file as an example for the configuration. Examples can be found in the DHT module. On this occasion a descriptive name for the file would be meaningful.